asset/css/markbind.css

```diff
@@ -198,28 +198,31 @@ so we apply temporary styles to fix the layout until Vue correctly updates the D
     display: none;
 }
 
-/* Hide the contents of unexpanded panels*/
-
+/* Hide the contents of unexpanded panels */
+/* stylelint-disable */
```

Comment:
You should use `/* stylelint-disable-next-line */`. Preferably with the specific rule disabled.

Likewise for the other lines below.

[Documentaton here](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#turning-rules-off-from-within-your-css)

Reply:
Ok, I disabled only the selector-type-no-unknown rule.